pom.xml

```diff
@@ -253,6 +254,8 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
                             <arg>-Xlint:all</arg>
                             <!-- Workaround for JDK bug https://bugs.openjdk.java.net/browse/JDK-6999068 -->
                             <arg>-Xlint:-processing</arg>
+                            <!-- Enable Java 9 compilation with 1.8 compatibility -->
+                            <arg>-Xlint:-options</arg>
```

Comment:
Where do we use the options? Is it the implicit `<target>${java.version}</target>`?

Reply:
this is needed because `java.version` is always `8`, even if the selected JDK is 9. The JDK cannot compile to another bytecode version, giving a warning. This disables the warning, since we're only checking the build, not producing any artifacts.
```
<java.version>1.8</java.version>
```